codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/rustlang/RustWriter.kt

```diff
@@ -202,6 +230,10 @@ fun <T : CodeWriter> T.documentShape(shape: Shape, model: Model, autoSuppressMis
     return this
 }
 
+fun RustWriter.superDocs(text: String, vararg args: Any): RustWriter {
```

Comment:
Can we rename superDoc to `crateLevelDoc` or `moduleDoc`?

Reply:
what about `container` doc`?